roles/docker/defaults/main.yml

```diff
@@ -6,3 +6,4 @@ docker_version: 1.9.1
 docker_rule_comment: "docker api"
 docker_device: ""
 docker_device_size: "10000MB"
+docker_device_metadata_size: "1000MB"
```

Comment:
so, FYI: ISTR LVM accepts percentages for size paramters, so you could try this:

```yaml
docker_device_size: "85%"
docker_device_metadata_size: "15%"
```

Which in theory should DWIW. It might be a better way of specifying this data for multiple entities on a disk than fixed sizes.